app/src/main/java/com/nervousfish/nervousfish/modules/util/AUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.nervousfish.nervousfish.modules.util;
+
+/**
+ * This class should be extended by all platform-specific {@link IUtils} implementations and contains
+ * all cross-platform functionality
+ */
+@SuppressWarnings("PMD.AbstractClassWithoutAnyMethod")
+abstract class AUtils implements IUtils {
```

Comment:
I would say every utility class should be added separately to the service locator 🙂